ORBIT/phases/install/cable_install/common.py

```diff
@@ -362,3 +362,31 @@ def pull_winch(vessel, distance, **kwargs):
         constraints=vessel.operational_limits,
         **kwargs,
     )
+
+@process
+def dig_trench(vessel, distance, **kwargs):
+    """
+    Task representing time required to dig a trench prior to cable lay and burial
+
+    Parameters
+    ----------
+    vessel : Vessel
+        Performing vessel. Requires configured `operational_limits`.
+    distance : int | float
+        Length of trench, equal to length of cable section (km).
+    trench_dig_speed : int | float
+        Speed at which trench is dug (km/hr).
+    """
+
+    key = "trench_dig_speed"
+    trench_dig_speed = kwargs.get(key, pt[key])
+    trench_dig_time = distance / trench_dig_speed
+    _vkwargs = getattr(vessel, "_transport_specs", {})
```

Comment:
Is the intention behind this so that you can define "trench_dig_speed" in "vessel.transport_specs"?

If so, you'd need to move this up to line 382 and combine kwargs prior to the kwargs.get() call.

Reply:
Let me know about this one - I can make a quick mod if you'd like.

Reply:
Hmm I just ran a quick test in jupyter and they weren't changing. The kwargs passed to vessel.task wouldn't change anything, just the 'trench_dig_time'. I may have messed this up when I was modifying some of these processes.

I just submitted a fix for all similar tasks in common.py.

Reply:
Hmm maybe it was just updating the label in the output logs and not the actual calculation when i ran it.  i'm good with your fix if you want to close this PR and merge everything in.